src/GitHub.VisualStudio/UI/Views/GitHubConnectContent.xaml.cs

```diff
@@ -101,7 +101,7 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
 
             var item = values[0] as ISimpleRepositoryModel;
             var context = values[1] as IGitAwareItem;
-            return context?.ActiveRepoUri == item?.CloneUrl && context?.ActiveRepo?.LocalPath == item?.LocalPath;
+            return context?.ActiveRepoUri == item?.CloneUrl && context?.ActiveRepo?.LocalPath.ToUpper() == item?.LocalPath.ToUpper();
```

Comment:
Yeah, my bad, my brain was obviously not working properly yesterday. Phil's approach is totally the correct one for this!